### ISSUE TITLE
Kotlin Multiplatform report is empty

### ISSUE THREAD

**Issue by JavierSegoviaCordoba** [Kotlin/kotlinx-kover#100]
Repro: https://github.com/Reproducers/kover-repro

Report:

```xml
<?xml version="1.0" ?>
<report name="Intellij Coverage Report">
<counter type="INSTRUCTION" missed="0" covered="0"/>
<counter type="BRANCH" missed="0" covered="0"/>
<counter type="LINE" missed="0" covered="0"/>
<counter type="METHOD" missed="0" covered="0"/>
<counter type="CLASS" missed="0" covered="0"/>
</report>
```

In the same repo, there is a JVM library that is working.

Both engines fail.

If I apply the plugin directly to the multiplatform subproject instead of in the root project, it works.

If I try to add kover to all subprojects, the report is empty in the multiplatform project.

```kotlin
buildscript {
    repositories {
        mavenCentral()
    }

    dependencies {
        classpath("org.jetbrains.kotlinx:kover:0.4.4")
    }
}

subprojects {
    pluginManager.apply("org.jetbrains.kotlinx.kover")
}
```

It is really weird behavior.

At the same time, everything I have tried is working in the JVM library.

**Comment by JavierSegoviaCordoba** [comment]
If I apply the plugin directly to the multiplatform subproject instead of in the root project, it works.

**Comment by shanshin** [comment]
Fixed in `0.5.0-RC`.
